net/pfSense-pkg-pfBlockerNG-devel/files/usr/local/pkg/pfblockerng/pfblockerng.inc

```diff
@@ -321,13 +321,22 @@ function pfb_filter_service() {
 
 	# Start pfBlockerNG Firewall filter Daemon
 	if [ -e '/var/log/filter.log' ]; then
-		/usr/local/sbin/clog_pfb -f /var/log/filter.log | /usr/local/bin/php_pfb -f /usr/local/pkg/pfblockerng/pfblockerng.inc filterlog &
+	# Decide about using clog or tail
+	if [ -f /usr/local/sbin/clog_pfb ]; then
+		tail_binary="/usr/local/sbin/clog_pfb -f"
+	else
+		tail_binary="/usr/bin/tail_pfb -F"
 	fi
 
+	${tail_binary} /var/log/filter.log \
+	    | /usr/local/bin/php_pfb \
+	    -f /usr/local/pkg/pfblockerng/pfblockerng.inc \
+	    filterlog &
```

Comment:
This block from line 324 until 334 must be indented with one more tab since it's inside another if statement